### PR TITLE
Support migrating and updating existing projects

### PR DESCRIPTION
**Pull request by llucax**
- Fix outdated documentation
- Overwrite existing files when renaming
- Fix typo in README
- Copy the replay file into the repo
- Add instructions to migrate and update existing projects

### CODE REVIEW

**Comment by llucax** [comment]
Based on #68 .

File: README.md [hunk 1]
```diff
@@ -55,3 +55,73 @@ python3 -m venv .venv
...
+cd ..
+cookiecutter gh:frequenz-floss/frequenz-repo-config-python \
+    --directory=cookiecutter \
+    --force \
+    --replay \
+    --replay-file project-directory/.cookiecutter-replay.json
+```
+
+This will create a new commit with all the changes to the overwritten files.
+Bear in mind that all the `TODO`s will come back, so there will be quite a bit
```
Comment (daniel-zullo-frequenz) [inline README.md:110]: s/Bear/Bare/

**Review by daniel-zullo-frequenz — COMMENTED** [review]
I just have a cosmetic comment that can be fixed in another PR as this was already merged

File: README.md (hunk 1 above)
Comment (shsms) [reply README.md:110]: I think bear is correct.  it is not the grizzly Bear,  it is like "I have to bear the weight", "I can't bear this", etc.

**Pull request merged**